src/v0/util/index.js

```diff
@@ -2108,6 +2108,28 @@ const parseConfigArray = (arr, key) => {
   return arr.map((item) => item[key]);
 };
 
+const GOOGLE_ALLOWED_CONSENT_STATUS = ['UNSPECIFIED', 'UNKNOWN', 'GRANTED', 'DENIED'];
```

Comment:
should this be placed in google specific utils rather than in common utils

Reply:
Also add unit test cases

Reply:
moved to util and added test cases